Simulation/Integrators/RK4_Integrator_Generator.m

```diff
@@ -0,0 +1,23 @@
+function Gen_RK4_Integrator = RK4_Integrator_Generator(dt)
+  
+  Gen_RK4_Integrator = @RK4_Integration;
+  
+  % 4th Order Runge-Kutta (RK4) integration
+  function next_state = RK4_Integration(Vh,stateVector = Vh.getStateVector(),controlsVector = Vh.getControlsVector)
```

Comment:
It doesn't seem like the `getControlsVector` function is being called here.

Reply:
Added parentheses to this, I think it may work either way but I like it to be not ambiguous.